src/lib/svg_text_utils.js

```diff
@@ -134,7 +134,7 @@ exports.convertToTspans = function(_context, gd, _callback) {
                 else if(svgClass[0] === 'l') {
                     newSvg.attr({x: _context.attr('x'), y: dy - (newSvgH / 2)});
                 }
-                else if(svgClass[0] === 'a') {
+                else if(svgClass[0] === 'a' && svgClass.indexOf('atitle') === -1) {
```

Comment:
Could we use 
<pre>
if(svgClass === 'atitle') {
</pre>
here?

Reply:
Why would that be better?

Reply:
I was wondering of an edge case where 'atitle' appears in other position in the text. 
Though I was not able find any word containing those letters in order!
Anyway could we use this condition instead to test if `atitle` only occurs at the start of the string?
<pre>
else if(svgClass[0] === 'a' && svgClass.indexOf('atitle') !== 0) {
</pre>

Reply:
Ok. Done in -> https://github.com/plotly/plotly.js/pull/3513/commits/c7878b381dc5822faf20570abfa74a814297e4b1